score-client/src/main/java/bio/overture/score/client/ssl/DumbHostnameVerifier.java

```diff
@@ -0,0 +1,12 @@
+package bio.overture.score.client.ssl;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLSession;
+
+public class DumbHostnameVerifier implements HostnameVerifier {
+  public DumbHostnameVerifier() {}
```

Comment:
We should find a more descriptive name than `Dumb` for this. `NoTestHostnameVerifier` perhaps?

Reply:
Have renamed this to NoTestHostnameVerifier and updated the packages for both score-core and score-server